### ISSUE TITLE
Questions Regarding the High Quality of Long Videos

### ISSUE THREAD

**Issue by BecauseImBatman0** [meituan-longcat/LongCat-Video#69]
Dear Long-Cat Video Team,

Thank you so much for open-sourcing such an outstanding model and technical report. I am incredibly impressed by Long-Cat Video's performance in long video generation; the quality is exceptionally high, with almost no degradation over time.

I'm particularly curious about this phenomenon, as it seems you haven't implemented any special treatments to combat degradation (such as self-forcing). In issue #8 , you mentioned that **"pretraining with video-continuation tasks" and "block-causal attention" are the primary factors.**

If I understand correctly, video-continuation tasks are a fundamental method for generating long videos by concatenating multiple segments. Does this imply that block-causal attention is the most important factor?

Furthermore, I have a more specific hypothesis: if the conditioning frames are clean (timestep = 0), is explicitly masking the attention to the noisy latents the most critical step for generating these high-quality long videos? I was also wondering if you have ever tried not applying an attention mask to the noisy latents, in a similar vein to other works that consider noise itself as a form of mask (i.e., "noise-as-mask").

I look forward to your reply. Thanks again for your remarkable work.

<img width="398" height="430" alt="Image" src="https://github.com/user-attachments/assets/b0af9394-8821-4d14-810d-03ccdcfea4c5" />

**Comment by BecauseImBatman0** [comment]
Thank you so much for your reply! Have a nice weekend : )
